docs/data-collection/scanning.md

```diff
@@ -598,6 +598,8 @@ It is critical to check that physiological signals are looking good:
 - [ ] Check the box confirming that it has been acquired in the issue collecting notes about the session.
     Don't forget to report any observations there.
 
+- [ ] Copy data from the ./data subfolder in the GitHub repository HCPh-fMRI-tasks into the synchronized Dropbox folder, and make sure the Wi-Fi is on for synchronization.
```

Comment:
```suggestion
- [ ] Copy data from the `./data` subfolder in the GitHub repository HCPh-fMRI-tasks into the synchronized Dropbox folder, and make sure the Wi-Fi is on for synchronization.
```

Reply:
Do Acknowledge and the Eye tracker save their data in the ./data subfolder in the HCPh-fMRI-tasks?
And then you have to copy them to the Dropbox?

Can you precise on which computer this happens?